server/flaskr/utils/input_validation.py

```
@@ -21,53 +32,95 @@ def verify_missing_inputs(data: dict, validators: list[InputValidator]) -> list[
         pass
 
-    def validate(self, data) -> bool:
+    def validate(self, data) -> InputError:
         id = data.get(self.field_name, None)
-        if id is None:
-            return False
+        if id is None or id == "":
+            return InputError.RequiredField
+        try:
+            obj = self.dataHandler.get_item_by_id(id)
+        except:
```

Ruff (0.14.14)

[error] Do not use bare `except`

(E722)